### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by JackHe313**
fixed the issue to display dates if trip in main dashboard 
implemented add user to trip

### CODE REVIEW

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 1]
```diff
@@ -88,6 +90,21 @@ public String validateInput(HttpServletRequest request) {
         }
         if(end.before(start))
             return "Start date should before end date";
+
+        String Pinput = StringEscapeUtils.escapeHtml4(request.getParameter(PARTICIPANTS_PARAM));
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java]: can you rename this to participantInput. It is clearer.

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 2]
```diff
@@ -100,14 +117,13 @@ public Trip getTrip(HttpServletRequest request){
...
         Date end = DataStoreHelper.parseInputDate(request.getParameter(END_DATE_PARAM));
-        //fix later
-        List<String> participants = new ArrayList<String>();
-        participants.add("1019264699");//jack
-        participants.add("-1939051496");//shaheen
-        participants.add("-1148042363");//emma
-        participants.add("-94026392");//Mauricio
-        participants.add("359360004");//YiFan
-        return Trip.create(tripID,textValuetitle,participants, totalBudget,start,end);
+        String Pinput = StringEscapeUtils.escapeHtml4(request.getParameter(PARTICIPANTS_PARAM));
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java]: rename this Pinput too. Variables should be cameCase. so participantInput

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 3]
```diff
@@ -100,14 +117,13 @@ public Trip getTrip(HttpServletRequest request){
...
+        List<String> IDS = new ArrayList<String>();
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java]: rename to participantIds. Uppercase is for consts

**Review by sbahaud — APPROVED** [review]
- Please update the PR description

Approving with a few minor comments

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 4]
```diff
@@ -52,7 +54,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             response.getWriter().println("Input information error: "+ error);
         }
 
-        response.sendRedirect("https://summer22-sps-36.appspot.com/");
+        //response.sendRedirect("/trip/tripDetails.html");
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java:60]: Should this be disabled?

Comment (JackHe313) [reply src/main/java/com/google/sps/app/servlets/NewTripServlet.java:60]: oh, I'll fix it back

Comment (JackHe313) [reply src/main/java/com/google/sps/app/servlets/NewTripServlet.java:60]: oh, it looks like this tripDetails.html isn't ready ,redirect to this html will cause error

**Review by xiaoalem — APPROVED** [review]

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 5]
```diff
@@ -44,15 +48,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+            response.getWriter().print(newTrip.tripID());
         } 
         else {
             System.out.println(request.getParameter(START_DATE_PARAM));
             System.out.println("Input information error: "+ error);
             response.getWriter().println("Input information error: "+ error);
         }
 
-        response.sendRedirect("https://summer22-sps-36.appspot.com/");
+        //response.sendRedirect("/trip/tripDetails.html");
```
Comment (emmayuan25) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java:60]: No need to fix now, but if added back, redirect to trip dashboard when trip successfully created

**Review by emmayuan25 — APPROVED** [review]
great work!

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java (hunk 5 above)
Comment (xiaoalem) [reply src/main/java/com/google/sps/app/servlets/NewTripServlet.java:60]: just deleted the redirect in Servlet, but added redirect to dashboard.html for now in tripEvent.js

**Pull request merged**